docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,28 @@
+import os.path
+from pathlib import Path
+import check_helper
+
+home = str(Path.home())
+exercise_commands = {'mkdir': 'creating directories', 'tree': 'viewing the directory tree'}
+exercise_paths = [home + '/test/tmp']
+
+def main():
+    '''
+    Running the tests and prompting the student according to his work.
+    '''
+    test_result = ''
+    student_commands = check_helper.get_student_commands()
```

Comment:
I think we can have another function, test_exercise or something similar, and all of what is in main could be there, and this function could be in the helper file as well, I believe you will use this function for other exercises as well.

Reply:
done